packages/Webkul/BookingProduct/src/Resources/views/shop/velocity/products/view/booking/slots.blade.php

```diff
@@ -7,7 +7,7 @@
             <label class="label-style required">{{ $title ?? __('bookingproduct::app.shop.products.book-an-appointment') }}</label>
 
             <div class="control-group-container">
-                <div class="form-group date" :class="[errors.has('booking[date]') ? 'has-error' : '']">
+                <div class="form-group date {{$product->locale == 'ar' ? 'rtl' : ''}}" :class="[errors.has('booking[date]') ? 'has-error' : '']">
```

Comment:
No need for a case here, as the body tag already has rtl. Handle it from scss in rtl file.